Source/Project64/Multilanguage/Language Class.cpp

```diff
@@ -315,7 +315,7 @@ void CLanguage::LoadDefaultStrings (void)
     DEF_STR(SAVE_FIRST_USED,     L"Use first-used save type");
     DEF_STR(SAVE_4K_EEPROM,      L"4-kbit EEPROM");
     DEF_STR(SAVE_16K_EEPROM,     L"16-kbit EEPROM");
-    DEF_STR(SAVE_SRAM,           L"32-kbyte SRAM");
+    DEF_STR(SAVE_SRAM,           L"SRAM");

```

Comment:
you forgot to update this in main english file too :P

https://github.com/project64/project64/blob/master/Lang/English.pj.Lang#L321